eng/common/scripts/add_pullrequest-labels.ps1

```diff
@@ -0,0 +1,46 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $true)]
+    $PRNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken,
+
+    [Parameter(Mandatory = $false)]
+    $PRLabel
```

Comment:
While I'm not opposed to having an independent script to add labels to a PR I would like for our [Submit-PullRequest](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/Submit-PullRequest.ps1) script to be able to take labels as an option. If we cannot do add the labels at the same time we create the PR then we can make the additional calls in to add the labels as part of that same script.

Reply:
I agreed to add labels in additional call to save the steps here. Will make the changes.